lidlplus/api.py

```diff
@@ -209,21 +209,16 @@ def _check_2fa_auth(self, browser, wait, verify_mode="phone", verify_token_func=
             browser.find_element(By.NAME, "VerificationCode").send_keys(verify_code)
             self._click(browser, (By.CLASS_NAME, "role_next"))
 
-    def login(self, phone, password, **kwargs):
+    def login(self, email, password, **kwargs):
```

Comment:
The login form now seems to require email instead of phone. Could you also replace all other occurences of this (e.g. in the login CLI) where there's "phone" to "email"?

I can see at least one occurence, where the variable should probably be named `email` and the `(phone number)` part of the string should be changed to `(email)`.
https://github.com/Andre0512/lidl-plus/blob/0f1cd92880fff43c67032709bc921840fd72ca2f/lidlplus/__main__.py#L87